pageserve/app.py

```diff
@@ -76,29 +76,19 @@ def parsed_posts(posts):
 
 def get_events():
     """Gets all sub-events from events service."""
-    # TODO(cmei4444): integrate with events service to pull event info from
-    # database
-    events = [{'event_id': '1',
-               'name': 'concert 1',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-10-2019',
-               },
-              {'event_id': '2',
-               'name': 'concert 2',
-               'description': 'listen to fun music here!',
-               'author': 'admin',
-               'created_at': '7-9-2019',
-               'event_time': '7-12-2019',
-               }]
-    return parsed_events(events)
-
-
-def parsed_events(events):
+    url = os.environ.get("EVENTS_ENDPOINT")
+    r = requests.get(url, params={})
```

Comment:
Does this work? It look like $EVENTS_ENDPOINT in Travis is missing the /v1/

Reply:
Thanks for the catch! Fixed in 0682de4